### PR TITLE
CLDR-13139 Upgrade Tomcat, more preliminary refactoring

### PR DESCRIPTION
**Pull request by btangmu**
-Move parts of server-playbook to new playbooks for mysql, nginx, tomcat, cldr-apps

##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13139
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by btangmu** [comment]
Something strange is happening with "Ansible Lint / build (pull_request) Failing after 2m — build"

https://github.com/unicode-org/cldr/pull/833/checks?check_run_id=1364801304

TASK [Setup index.html] ********************************************************
fatal: [surveytool]: FAILED! => {"changed": false, "checksum": "f4c9c041b6d569d8719544c6e59158d973989381", "msg": "Destination directory /var/www does not exist"}

Something wrong with Vagrant config? No /var/www?

**Comment by srl295** [comment]
Does it work locally?

With `vagrant up`

**Comment by srl295** [comment]
I'll try this locally

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
@btangmu I'm seeing that failure happen without any related changes in https://github.com/unicode-org/cldr/actions/runs/354250196

File: tools/scripts/ansible/setup-playbook.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 - import_playbook: server-playbook.yml
+- import_playbook: mysql-playbook.yml
+- import_playbook: nginx-playbook.yml
+- import_playbook: tomcat-playbook.yml
+- import_playbook: cldr-apps-playbook.yml
 - import_playbook: packages-playbook.yml
 - import_playbook: certbot-playbook.yml
 - import_playbook: hostname-playbook.yml
```
Comment (srl295) [inline tools/scripts/ansible/setup-playbook.yml:8]: @btangmu it's the order of execution.  You have 'server playbook` before `nginx playbook`, so it tries to update `/var/www/` before nginx is actually installed.

So this is a valid error. putting `nginx-playbook.yml` first seemed  to help, but please consider the order

**Review by srl295 — CHANGES_REQUESTED** [review]

File: tools/scripts/ansible/setup-playbook.yml (hunk 1 above)
Comment (btangmu) [reply tools/scripts/ansible/setup-playbook.yml:8]: Thanks! I'll work on it

**Comment by btangmu** [comment]
See https://github.com/unicode-org/cldr/pull/839

**Pull request closed without merging**